cmd/gnmi_capabilities/gnmi_capabilities.go

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2017 Google Inc.
```

Comment:
add NTT copyright

Reply:
also, it would be good to put a line saying from where you got this example.

https://github.com/google/gnxi/blob/master/gnmi_capabilities/gnmi_capabilities.go